doc/source/ray-observability/state/state-api.rst

```diff
@@ -482,6 +487,27 @@ E.g., Stream a log file from a node
         for line in get_log(filename="raylet.out", node_ip=<NODE_IP>, follow=True):
             print(line)
 
+E.g., Stream log from an actor with actor id
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. tabbed:: CLI
+
+    .. code-block:: bash
+
+        ray logs actor --id=<ACTOR_ID> --follow
+
+.. tabbed:: Python SDK
+
+    .. code-block:: python
+
+        from ray.experimental.state.api import get_log
+
+        # Node IP could be retrieved from list_nodes() or ray.nodes()
+        # You could get the actor's ID from the actor's handle, e.g.
+        # `your_actor._actor_id.hex()` or with `ray list actors`.
```

Comment:
Hmm, `_actor_id` is not a public method, do we have a public method here?

Reply:
Hmm, I don't think there is. Let me maybe rephrase this as of now.

Reply:
Asked in the core team and see if we want to add such util function.

Reply:
Yes, to make this really effective, and integrate state observability with the Ray API, I think a public method like this would be very useful.